### DISCUSSION TITLE
Are scrolling comments during replays (nicovideo style) ever being added back?

### DISCUSSION THREAD

**Discussion by StalkyBoi43815** [ppy/osu#28448] [Ideas] [answered]
That would be really awesome, but I can totally see why they were removed.

**Comment by BlockHead007** [comment, marked as answer]
Requested already: #4796

**Reply by bdach** [reply]
Mind the fact that that issue is years old and I believe since then the scrolling comments were shut down on stable because of moderation concerns, so I dunno if that issue is still up to date wrt conclusion and the like.

**Reply by BlockHead007** [reply]
> scrolling comments were shut down on stable because of moderation concerns

Were they? I just managed to leave a comment in latest stable build. Restarted replay and it is on beatmap or it's invincible for others?

**Reply by bdach** [reply]
I sort of misremembered it, there *were* plans of shutting those down because of moderation concerns, but they might have never actually come to fruition because of other priorities. Take that for what you will though with respect to the future of this feature.

**Reply by peppy** [reply]
I thought it was disabled too
